test/unit/LicenceYearTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace SendGrid\Test;
+
+class LicenceYearTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstructor()
+    {
+        $license = explode("\n", file_get_contents('./LICENSE.txt'));
```

Comment:
In the previous build on Travis, the test was failing because it couldn't find this file. I am wondering if we are running into a file path issue? 
The test is running from a different directory than the License file, should we be using a path relative to the file system, rather than the repo, instead?